README.md

```diff
@@ -10,7 +10,7 @@ DDEV integration for developing Drupal contrib projects. As a general philosophy
 1. If you haven't already, [install Docker and DDEV](https://ddev.readthedocs.io/en/latest/users/install/)
 2. `git clone` your contrib module
 3. cd [contrib module directory]
-4. Configure DDEV for Drupal using `ddev config --project-name=[contrib module] --project-type=drupal --docroot=web  --php-version=8.2` or select these options when prompted using `ddev config`
+4. Configure DDEV for Drupal using `ddev config --project-type=drupal --docroot=web  --php-version=8.2 --project-name=[contrib module]` or select these options when prompted using `ddev config`
```

Comment:
Just a note that `--project-name` is not that useful here anyway. Why not just omit it? Then it will get the project name of the enclosing directory, which is what was expected anyway. Then no copy/paste required.

Reply:
```suggestion
4. Configure DDEV for Drupal using `ddev config --project-type=drupal --docroot=web  --php-version=8.2]` or select these options when prompted using `ddev config`
```

Reply:
I'm also not sure why `php-version` is specified here.

Reply:
Honestly my understanding is those are here to:
- Underline that these are available options, for those who need to control them.
- Project name is important if your enclosing folder contains `_`, which is very common for drupal contrib modules.

Reply:
Project name is no longer important from [DDEV v1.23.4](https://github.com/ddev/ddev/releases/tag/v1.23.4), since underscores are automatically replaced with hyphens in project name.

Underlining that the options are available is indeed a good teaching technique.

Reply:
Good to know that! I will resolve the conv, as I think it is important to leave the other options.